scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -123,8 +124,32 @@ <h5 class="header">General Institute Settings</h5>
       </div>
     </div>
   </div> <!-- end of div class="row" -->
-</div> <!-- end of container div -->
+
+</form>
+
+<!-- Display form to add a Beacon dataset if user is an andmin and if institute lacks one or more Beaocn datasets -->
+{% if current_user.is_admin and config.BEACON_URL and config.BEACON_TOKEN %}
+<form action="{{ url_for('overview.add_beacon_dataset', institute_id=institute._id) }}" class="form-inline" method="post">
```

Comment:
`form-inline` has been dropped in bs5. Use grid or utils instead! Which you already do, so just skip! 😁